config/composer.json

```diff
@@ -34,6 +34,9 @@
     },
     "extra": {
         "patches": {
+            "mglaman/drupal-check": {
+                "Invalid error for yaml plugin managers": "patches/drupal-check_yaml-plugin.patch"
```

Comment:
Can you add a link to your open issue ticket?

Reply:
:heavy_check_mark: